Lib/Rabbit4000/tcpip/DMAETH100.LIB

```diff
@@ -1584,25 +1653,20 @@ int dmaeth100_receive(_DMAEth100Config * nic)
       call dmaeth100_alloc_rx
       jr    c,.cont1
 
+		/* Don't have to do this with the R6000...tx fire will do it */
 #if _RAB6K
-		ld		hl,(ix+[_decb]+rxbit)
-		ex		de,hl
-		ioi	ld	hl,(DMCSLR)
-		and	hl,de
+		;ld		hl,(ix+[_decb]+rxbit)
+		;ex		de,hl
+		;ioi	ld	hl,(DMCSLR)
+		;and	hl,de
 #else
       ld		a, (ix+[_decb]+rxbit)
       ld		b,a
       ioi	ld a,(DMCSR)
       and   a,b
 #endif
-      jr		nz,.cont1
 
-      ;FIXME - this should not be required!

```

Comment:
Ya, this was me leaving that code in because there was very limited explanation of why it was included in the first place.  The R6000 with the single rx buffer didn't need it.  The R5000 with that two rx buffer "might" need it, but it seemed like whoever coded that didn't understand why either (hence the FIXME in there by someone else.)

I left it just in case somebody wanted to go figure out the two buffer scenario with this codebase.  The "right" fix is probably a pool of buffers for best performance  ;-)